package.json

```diff
@@ -1,17 +1,23 @@
 {
   "name": "eslint-config-mavenlint",
-  "version": "1.0.3",
+  "version": "1.1.0",
   "description": "Mavenlink ESLint config",
   "main": "index.js",
   "repository": "https://github.com/mavenlink/mavenlint.git",
   "license": "MIT",
-  "peerDependencies": {
-    "eslint": ">= 3.7.0"
+  "devDependencies": {
+    "eslint": "^3.7.0",
+    "eslint-plugin-import": "^1.13.0",
+    "eslint-plugin-jsx-a11y": "^2.1.0",
+    "eslint-plugin-react": "^6.1.2"
   },
-  "dependencies": {
-    "eslint-config-airbnb": "^10.0.1",
+  "peerDependencies": {
+    "eslint": "^3.7.0",
```

Comment:
Which project is complaining?  And what version of eslint does that project install?  I've usually seen the missing peer dep warning in three cases:  when I've screwed up my dependencies, when the library has screwed up its dependencies, and when there's an install ordering issue.

The third case would happen if this package gets installed before eslint. It won't see eslint yet, so will complain, but there isn't actually an issue.  I'm not sure which one is the problem here, but the `>= 3.7.0` should work fine based on my experience.